src/components/country-entry-card/country-entry-card.js

```diff
@@ -0,0 +1,21 @@
+
+import React from 'react';
+import { connect } from 'react-redux';
+import * as actions from 'actions/url-actions';
+import Component from './country-entry-card-component';
+
+const ToggleSceneWidget = props => {
```

Comment:
Someone copied something here :smile:
```suggestion
const CountryEntryCard = props => {
```

Reply:
😅